apps/www/content/docs/forms/react-hook-form.mdx

```diff
@@ -142,6 +142,8 @@ export function ProfileForm() {
 }
 ```
 
+Note that default values for each field are required. Without them, they will be [uncontrolled components](https://react.dev/reference/react-dom/components/input#controlling-an-input-with-a-state-variable), and React will error.
```

Comment:
This is to be consistent with the note at line 87. Feel free to let me know if you'd prefer not to receive PRs of small importance ;-)

Reply:
Can we rephrase this to:

Since FormField is using a controlled component, you need to provide a default value for  the field. See the [React Hook Form docs](https://react-hook-form.com/api/usecontroller) to learn more about controlled components.

Reply:
Of course! Done ;-)